homework_36/src/main/java/org/spring_framework/service/LoggingServiceImpl.java

```diff
@@ -0,0 +1,15 @@
+package org.spring_framework.service;
+
+import org.springframework.stereotype.Service;
+
+import java.util.logging.Logger;
+
+@Service
+public class LoggingServiceImpl implements LoggingService {
+    private static final Logger LOGGER = Logger.getLogger(LoggingServiceImpl.class.getName());
```

Comment:
It is good that you decided to extract all logging in a separate service. But, a logger should be declared per class. The reason is that you should pass exactly that class name whose method is called, because if it throws an exception you should log exactly the same method where it was thrown. In case of this custom logging service you cannot reach that. So, remove it and use logger per class approach.